R/verb-arrange.R

```diff
@@ -1,3 +1,28 @@
+#' Arrange rows by variables in a local lazy tibble
+#'
+#' Order tbl_lazy rows by an expression involving its variables.
+#'
+#' @section Missing values:
+#' When applied to `tbl_lazy`, the [arrange()] method sorts NAs first (at the
```

Comment:
I think this needs some caveat about "most databases"

Reply:
Docs now [say for "most database tables"](https://github.com/tidyverse/dbplyr/pull/222/commits/6873d6482a1cfa076b4d49d0787485ab996cc3cd#diff-12fbbf8bf4196bb24a276dff57ec3116R7).